platform/kube/BUILD

```diff
@@ -43,6 +43,7 @@ filegroup(
 
 go_test(
     name = "go_default_test",
+    size = "small",
```

Comment:
According to https://bazel.build/versions/master/docs/test-encyclopedia.html size="small" implies timeout of 60 seconds.